src/GitHub.Api/Events/RepositoryWatcher.cs

```diff
@@ -388,5 +443,26 @@ public void Dispose()
         }
 
         protected static ILogging Logger { get; } = Logging.GetLogger<RepositoryWatcher>();
+
+        private enum EventType
+        {
+            None,
+            ConfigChanged,
+            HeadChanged,
+            RepositoryChanged,
+            IndexChanged,
+            RemoteBranchDeleted,
+            RemoteBranchCreated,
+            RemoteBranchChanged,
```

Comment:
It's odd...
Remote branches never change... they only seem to delete and recreate...

Maybe we should remove this enum entry and that explains that fact?

Reply:
Hmmm, good point. Maybe if the branch gets deleted and created again, we should raise it as a change?